### PR TITLE
Interaction branch - example code modified and added for click

### PR DESCRIPTION
**Pull request by rpallavisharma**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Description

modified code sample to show selenium dev example site and added code for click
### Motivation and Context

the example code needed update and for click no code was there
### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/webdriver/elements/interactions.en.md [hunk 1]
```diff
@@ -54,98 +110,79 @@ possible keystrokes that WebDriver Supports.
...
+	driver.find_element(name: 'email_input').send_keys 'admin@localhost.dev'
 
-ensure
-  driver.quit
-end
   {{< /tab >}}
   {{< tab header="JavaScript" >}}
-const {Builder, By, Key} = require('selenium-webdriver');
 
-(async function example() {
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/elements/interactions.en.md:117]: PR Looks good to me!

But the main idea is to move code samples to examples folder and render it from there!

Comment (rpallavisharma) [reply website_and_docs/content/documentation/webdriver/elements/interactions.en.md:117]: i don't know what that means. explain that, and i will then make the changes there. @harsha509

Comment (rpallavisharma) [reply website_and_docs/content/documentation/webdriver/elements/interactions.en.md:117]: Hi @harsha509 , can we accept this PR, as there are two components of change with respect to documentation - 1 is to update examples to the web pages listed on selenium.dev and another is to add in code files. let me know. thanks.

**Comment by harsha509** [comment]
Sure @pallavigitwork ,

but later the code samples should be moved to GitHub as discussed in slack!

**Comment by rpallavisharma** [comment]
Thanks @harsha509 .

**Pull request merged**